chunks/tool-yugabytedb/Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN curl -sSLo ./yugabyte.tar.gz https://downloads.yugabyte.com/releases/${YB_VE
 ENV PATH="$HOME/.pyenv/bin:$HOME/.pyenv/shims:$PATH"
 ENV PYENV_ROOT="$HOME/.pyenv"
 RUN git clone https://github.com/pyenv/pyenv.git ~/.pyenv \
-	&& git -C ~/.pyenv checkout ff93c58babd813066bf2d64d004a5cee33c0f27b \
+	&& git -C ~/.pyenv checkout 9ecd803bffaffb949fbdd8c70cb086227f6a3202 # v2.4.10 \
```

Comment:
<img width="947" alt="image" src="https://github.com/user-attachments/assets/da76c210-32d9-41fa-9be6-5f5a3d23de70">

Maybe `\` will be ignored